robot/src/main/kotlin/org/team9432/robot/auto/AutoConstants.kt

```diff
@@ -0,0 +1,47 @@
+package org.team9432.robot.auto
+
+import edu.wpi.first.math.geometry.Pose2d
+import edu.wpi.first.math.geometry.Rotation2d
+import edu.wpi.first.math.geometry.Transform2d
+import edu.wpi.first.math.geometry.Translation2d
+import org.team9432.robot.FieldConstants
+import org.team9432.robot.subsystems.RobotPosition
+import kotlin.math.pow
+import kotlin.math.sqrt
+
+object AutoConstants {
+    val fourNoteFirstShotPose = Pose2d(2.359, 4.418, Rotation2d.fromDegrees(152.5214))
```

Comment:
How are we getting these coordinates? Are they in meters? Where is the origin? Comments would be great here.